test/check-terminal

```diff
@@ -51,32 +51,6 @@ class TestTerminal(MachineCase):
         b.key_press([ 'e', 'c', 'h', 'o', ' ',  'ä', 'ö', 'ü', 'Return' ])
         wait_line(4, u'äöü')
 
-    def testReset(self):
-        b = self.browser;
-        m = self.machine;
-        b.default_user = "admin"
-
-        self.login_and_go("/local/terminal")
-
-        def line_sel(i):
-            return '#terminal .terminal div:nth-child(%d)' % i
-
-        def line_text(t):
-            return t + u'\xa0'*(80-len(t))
-
-        def wait_line(i,t):
-            b.wait_text(line_sel(i), line_text(t))
-
-        b.focus('#terminal .terminal')
-
-        # wait for prompt in first line and remember it
-        b.wait_text_not(line_sel(1), line_text(""))
-        prompt = b.text(line_sel(1))
-
-        # Run some commands
-        b.key_press( [ 'w', 'h', 'o', 'a', 'm', 'i', 'Return' ] )
-        wait_line(2, "admin")
-
```

Comment:
Argh, sorry, I totally missed that this isn't the whole function but that there is more stuff coming that you didn't delete.